### PR TITLE
Softsign basic implementation according to mindspore docs

### PR DESCRIPTION
**Pull request by vmurari17**
Tried implementing #17758  for #14951 as an applicant task. I'm new to open source contribution & hence if changes are required please let me know.

### CODE REVIEW

**Review by jkeane508 — COMMENTED** [review]
Hey!

Should there not be a test alongside this?

**Comment by vmurari17** [comment]
Hi there @jkeane508 ,
I had written a test function but couldn't make it work with the frontend even after going through the youtube tutorial of ivy testing and I am sure as of front end logic this snippet would be working. I'll share my test code as comment, would you be able to guide me a bit on that as of why the tests fail/don't work ?

**Comment by jkeane508** [comment]
> Hi there @jkeane508 , I had written a test function but couldn't make it work with the frontend even after going through the youtube tutorial of ivy testing and I am sure as of front end logic this snippet would be working. I'll share my test code as comment, would you be able to guide me a bit on that as of why the tests fail/don't work ?

Hey, I would add this test in anyways. Because we will need a comprehensive testing suite for this frontend regardless.

**Comment by vmurari17** [comment]
Hey @jkeane508 , I have added a test by taking the other frontend tests as an example, please let me know how can I further add up to this PR, appreciate your help

File: ivy/functional/frontends/mindspore/ops/function/nn_func.py [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+# local
+import ivy
+from ivy.func_wrapper import with_supported_dtypes
+from ivy.functional.frontends.paddle.func_wrapper import to_ivy_arrays_and_back
+
+
+@with_supported_dtypes({"2.0 and below": ("float16", "float32")}, "mindspore")
+@to_ivy_arrays_and_back
+def softsign(x):
+    """
```
Comment (jkeane508) [inline ivy/functional/frontends/mindspore/ops/function/nn_func.py:13]: I don't think we need to include docstrings in the frontend as users wont really interact with these functions.

**Review by jkeane508 — CHANGES_REQUESTED** [review]

**Comment by jkeane508** [comment]
ivy-gardener

File: ivy/functional/frontends/mindspore/ops/function/nn_func.py (hunk 1 above)
Comment (vmurari17) [reply ivy/functional/frontends/mindspore/ops/function/nn_func.py]: Alright, apologies for the impractical implementation, I have removed the docstrings in my latest commit

**Comment by vmurari17** [comment]
@jkeane508 What's the next step for this PR ?

**Review by jkeane508 — APPROVED** [review]
LGTM!

**Pull request merged**